web/client/selectors/maps.js

```diff
@@ -24,6 +24,8 @@ const mapDescriptionSelector = (state, id) => mapFromIdSelector(state, id) && ma
 const mapDetailsUriFromIdSelector = (state, id) => mapFromIdSelector(state, id) && mapFromIdSelector(state, id).details || "";
 const mapPermissionsFromIdSelector = (state, id) => mapFromIdSelector(state, id) && mapFromIdSelector(state, id).permissions || "";
 const mapThumbnailsUriFromIdSelector = (state, id) => mapFromIdSelector(state, id) && mapFromIdSelector(state, id).thumbnail || "";
+const hasMapAccessLoadingError = (state) => get(state, state.maps && "maps.loadingError");
```

Comment:
get already checks if some of the elements in the path are null, so you should simply use get(state, "maps.loadingError")

Reply:
I think loadingError is in `mapinitialConfig` state. Isn't it? did you tried this?